core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -525,6 +525,9 @@
   TSERV_SLOW_FLUSH_MILLIS("tserver.slow.flush.time", "100ms", PropertyType.TIMEDURATION,
       "If a flush to the write-ahead log takes longer than this period of time,"
           + " debugging information will written, and may result in a log rollover."),
+  TSERV_SLOW_FILEPERMIT_MILLIS("tserver.slow.filepermit.time", "100ms", PropertyType.TIMEDURATION,
```

Comment:
A larger default may be good.  If its too low it will fire too often and will be ignored.  Maybe at least one second?  or 500ms?

Reply:
I would think that a system that is blocking at all waiting for file permits would want to know since it ends up being rolled into the scan time. Since proper configuration of the tserver readahead/maxfiles open and the various compaction settings for files/ratios will prevent this log message from ever firing. I'm happy to make the default whatever value you prefer though.

Reply:
> I would think that a system that is blocking at all waiting for file permits would want to know since it ends up being rolled into the scan time.

Reading that made me think it may be nice to have this time as a metric.

Reply:
> Since proper configuration of the tserver readahead/maxfiles open and the various compaction settings for files/ratios will prevent this log message from ever firing.

I was thinking it may happen occasionally under normal circumstances, but its not a concern unless it too long.  But what is too long,  I don't know.  The thinking that it should never happen is a valid view point and may suggest a lower default value.  Now I am not sure what the default should be, because I see merit in both view points.  This makes me like the idea of a metric a bit more.

Reply:
Also, I am fine with leaving at 100ms.  That time seems like a good default for we don't expect this to ever happen, but if it does its ok as long as its not too long.